playground/README.md

```diff
@@ -1,69 +1,42 @@
-<div align="center">
+# BDK playground
 
-  <h1><code>wasm-pack-template</code></h1>
+The playground is a porting of certain functions of `bdk-cli` over wasm32,
+deployed at https://bitcoindevkit.org/bdk-cli/playground/
 
-  <strong>A template for kick starting a Rust and WebAssembly project using <a href="https://github.com/rustwasm/wasm-pack">wasm-pack</a>.</strong>
+## Compiling and running
 
-  <p>
-    <a href="https://travis-ci.org/rustwasm/wasm-pack-template"><img src="https://img.shields.io/travis/rustwasm/wasm-pack-template.svg?style=flat-square" alt="Build Status" /></a>
-  </p>
-
-  <h3>
-    <a href="https://rustwasm.github.io/docs/wasm-pack/tutorials/npm-browser-packages/index.html">Tutorial</a>
-    <span> | </span>
-    <a href="https://discordapp.com/channels/442252698964721669/443151097398296587">Chat</a>
-  </h3>
-
-  <sub>Built with 🦀🕸 by <a href="https://rustwasm.github.io/">The Rust and WebAssembly Working Group</a></sub>
-</div>
-
-## About
-
-[**📚 Read this template tutorial! 📚**][template-docs]
-
-This template is designed for compiling Rust libraries into WebAssembly and
-publishing the resulting package to NPM.
-
-Be sure to check out [other `wasm-pack` tutorials online][tutorials] for other
-templates and usages of `wasm-pack`.
-
-[tutorials]: https://rustwasm.github.io/docs/wasm-pack/tutorials/index.html
-[template-docs]: https://rustwasm.github.io/docs/wasm-pack/tutorials/npm-browser-packages/index.html
-
-## 🚴 Usage
-
-### 🐑 Use `cargo generate` to Clone this Template
-
-[Learn more about `cargo generate` here.](https://github.com/ashleygwilliams/cargo-generate)
+Update the [bdk-cli](./bdk-cli) submodule:
 
 ```
-cargo generate --git https://github.com/rustwasm/wasm-pack-template.git --name my-project
-cd my-project
+git submodule update --init --recursive
 ```
 
-### 🛠️ Build with `wasm-pack build`
+Compile bdk-cli using the right features and the WASM target:
 
 ```
-wasm-pack build
+cd bdk-cli
+cargo build --no-default-features --features esplora-reqwest,async-interface,compiler --target wasm32-unknown-unknown
 ```
 
-### 🔬 Test in Headless Browsers with `wasm-pack test`
+From inside the `bdk-cli` directory, run `wasm-bindgen`
 
 ```
-wasm-pack test --headless --firefox
+wasm-bindgen --target bundler --out-dir target/wasm32-unknown-unknown/debug/ target/wasm32-unknown-unknown/debug/bdk-cli.wasm
```

Comment:
You should make a release build, for size and performance reasons

Reply:
I think it's already a release build, as there's no `--debug`: https://rustwasm.github.io/wasm-bindgen/reference/cli.html

Reply:
If cargo is placing the wasm file in target/debug then it's probably not a release build

Reply:
Oh, I thought you meant the wasm-bindgen part. Updated and re-uploaded the autogenerated files, thanks!